examples/pytorch/eeg-gcnn/main.py

```diff
@@ -13,6 +15,14 @@
 from sklearn.model_selection import train_test_split
 from torch.utils.data import WeightedRandomSampler
 
+
+def load_memory_mapped_array(file_name):
+    # Since `file_name` is not always byte-aligned,
+    # let's fix this to avoid the warning.
```

Comment:
Sorry, I can't find a good explanation why ` joblib.load + joblib.dump` solves this problem. It looks like `joblib.load` is able to read unaligned files and `joblib.dump` and `joblib.dump` always creates properly aligned files.

From what I can see in debugger, the initial size of file "psd_features_data_X" is 86528478. After 
```
       joblib.dump(joblib.load(file_name), file_name)
```
it becomes 86528481. The values of the elements of the mapped arrays `x` and `y`  are the same in both cases:

- before my changes:
 ```
       x = load(memmap_x, mmap_mode="r")
       y = load(memmap_y mmap_mode="r")
 ```
- with my changes:
 ```
       x = load_memory_mapped_array(f"psd_features_data_X")
       y = load_memory_mapped_array(f"labels_y")
```

I am not sure, just another thought... Perhaps `psd_features_data_X` was created with some previous version of `joblib.dump` that had the memory alignment bug mentioned in the [Release 1.2.0](https://joblib.readthedocs.io/en/latest/developing.html) notes:

- **Fix memory alignment bug for pickles containing numpy arrays. This is especially important when loading the pickle with mmap_mode != None as the resulting numpy.memmap object would not be able to correct the misalignment without performing a memory copy. This bug would cause invalid computation and segmentation faults with native code that would directly access the underlying data buffer of a numpy array, for instance C/C++/Cython code compiled with older GCC versions or some old OpenBLAS written in platform specific assembly.**

Reply:
I see, thanks for the explanation, looks like the root cause is the original data prepared by the example is slightly polluted. Let's comment it here, see my recommendation blow.